sdk/storage/azure-storage-blobs/test/ut/block_blob_client_test.cpp

```diff
@@ -207,7 +207,7 @@ namespace Azure { namespace Storage { namespace Test {
   {
     auto blobClient = m_blobContainerClient->GetBlobClient(RandomString());
     auto res = blobClient.StartCopyFromUri(m_blockBlobClient->GetUrl());
-    EXPECT_NE(res.GetRawResponse(), nullptr);
+    EXPECT_EQ(res.GetRawResponse().GetStatusCode(), Azure::Core::Http::HttpStatusCode::Accepted);
```

Comment:
clever

Reply:
Thanks, I wanted to do something to guarantee the object was real when returned.  :)